ci/scripts/common.sh

```diff
@@ -1,6 +1,5 @@
 export CARGO_TERM_COLOR=always
 export PROTOC_NO_VENDOR=true
-export CARGO_HOME=/risingwave/.cargo
```

Comment:
Introduced with cargo registry cache in #3297, which was already removed in #9619.